recipes/itk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class HelloTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    version = "4.13.2"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def requirements(self):
+        self.requires("itk/{0}".format(self.version))
+
```

Comment:
There is no need to add `requirements` here, Conan will do it for the `test_package/conanfile`

Reply:
done